test/trajectory-subsetter/TrajectorySubsetter_Regression.ipynb

```diff
@@ -402,7 +407,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.12"
+   "version": "3.11.5"
```

Comment:
Same question here

Reply:
same answer, just metadata, the version of python used when the notebook was edited.